buildkite/pipeline.yml

```diff
@@ -33,3 +33,10 @@ steps:
     branches: "master"
     build:
       branch: master
+
+  - trigger: "orderly-dot-server-test"
+    label: ":rocket: orderly.server (from orderly) :cloud:"
+    build:
+      branch: !master
```

Comment:
quoting this causes it to interpret `!master` as the branch name instead of as not `master`